apps/web/modules/insights/insights-virtual-queues-view.tsx

```diff
@@ -40,7 +40,12 @@ export default function InsightsVirtualQueuesPage() {
       />
       <div className="mt-10">
         {selectedForm ? (
-          <TestForm form={selectedForm} supportsTeamMembersMatchingLogic={true} showAllData={false} />
+          <TestForm
+            form={selectedForm}
+            supportsTeamMembersMatchingLogic={true}
+            showAllData={false}
```

Comment:
`showAllData` isn't used  inside TestForm. Should we remove it or fix it?

Reply:
Will remove - i don't think we need an option to hide it here anymore